perf_test/blas/blas3/KokkosBlas3_gemm_perf_test.hpp

```diff
@@ -375,36 +406,59 @@ template <class TransAType, class TransBType, class BlockingType, class AlgoTag,
 void __do_gemm_parallel_batched_template(options_t options,
                                          gemm_args_t gemm_args) {
   using execution_space = typename device_type::execution_space;
+#if defined(KOKKOSKERNELS_GEMM_PERF_TEST_USE_RANGE_POLICY)
+  printf("Using RangePolicy!\n");
+  using policy_type     = Kokkos::RangePolicy<execution_space>;
+  using functor_type =
+      parallel_batched_gemm_range_policy<TransAType, TransBType, BlockingType>;
+#else
   using policy_type     = Kokkos::TeamPolicy<AlgoTag, execution_space>;
```

Comment:
TODO: Move AlgoTag=Serial out of this function and into another function that uses RangePolicy